src/lib.rs

```diff
@@ -89,6 +89,7 @@ const MODEL_WHITELIST: &[&str] = &[
     "serw12",
     "thelio-b1",
     "thelio-b2",
+    "thelio-b3",
```

Comment:
Strongly recommend updating the version in Cargo.toml, running cargo update and checking in Cargo.lock, and adding a new debian/changelog entry